src/Pyz/Zed/ContactUs/Persistence/ContactUsRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pyz\Zed\ContactUs\Persistence;
+
+use Spryker\Zed\Kernel\Persistence\AbstractRepository;
+use Generated\Shared\Transfer\ContactUsTransfer;
+
+
+/**
+ * @method \Pyz\Zed\ContactUs\Persistence\ContactUsPersistenceFactory getFactory()
+ */
+class ContactUsRepository extends AbstractRepository implements ContactUsRepositoryInterface
+{
+    /**
+     *
+     * @return \Orm\Zed\ContactUs\Persistence\PyzContactUs[]
```

Comment:
return typehint is misleading, you return an array of `ContactUsTransfer` transfers.